test/api/support/DownstreamResponseMappingSupportSpec.scala

```diff
@@ -59,6 +60,13 @@ class DownstreamResponseMappingSupportSpec extends UnitSpec {
       }
     }
 
+    "ifs returns UNMATCHED_STUB_ERROR" must {
```

Comment:
Just thought something actually. I don't think this is necessarily accurate because ifs wouldn't send back UNMATCHED_STUB_ERROR, it would only be our stub, right? As this error is only available in stubbed environments. So wouldn't it make more sense for it to say just downstream or 'stubinstead of ifs

Reply:
'Downstream' seems to be more accurate, will update now